sdk/identity/identity/review/identity.api.md

```diff
@@ -266,9 +267,6 @@ export interface ManagedIdentityCredentialClientIdOptions extends TokenCredentia
     clientId?: string;
 }
 
-// @public
-export type ManagedIdentityCredentialOptions = ManagedIdentityCredentialClientIdOptions | ManagedIdentityCredentialResourceIdOptions;
```

Comment:
I believe this is a breaking change 🤔

Reply:
Oh I thought it would be, but it’s not part of 2.0.4, so it is not in fact a breaking change! https://github.com/Azure/azure-sdk-for-js/blob/%40azure/identity_2.0.4/sdk/identity/identity/review/identity.api.md yay